docs/content/guides/developer/sui-101/sign-and-send-txn.mdx

```diff
@@ -0,0 +1,249 @@
+
+    // verify the signature against the transaction locally. if this fails the transaction probably will fail submitting. 
+    let res = sui_sig.verify_secure(&intent_msg, sender, sui_types::crypto::SignatureScheme::ED25519);
+    assert!(res.is_ok());
+```
+
+Finally, submit the transaction with the signature. 
+
+```Rust
+    let transaction_response = sui_client
+        .quorum_driver_api()
+        .execute_transaction_block(
+            sui_types::transaction::Transaction::from_generic_sig_data(
+                intent_msg.value,
+                Intent::sui_transaction(),
+                vec![GenericSignature::Signature(sui_sig)],
+            ),
+            SuiTransactionBlockResponseOptions::default(),
+            None,
+        )
+        .await?;
+
+```
+
+</TabItem>
+
+<TabItem value="cli" label="Sui CLI">
+
+The Sui CLI initiates a local file in `~/.sui/keystore` on your machine with a list of private keys (encoded as Base64 encoded `flag || 32-byte-private-key`) that you can use to sign transactions by specifying its address.
+
+There are three ways to initialize a key:
+
+```shell
+# generate randomly. 
+sui client new-address ed25519
+sui client new-address secp256k1
+sui client new-address secp256r1
+
+# import the 32-byte private key to keystore.
+sui keytool import "0xd463e11c7915945e86ac2b72d88b8190cfad8ff7b48e7eb892c275a5cf0a3e82" ed25519
+sui keytool import "0xd463e11c7915945e86ac2b72d88b8190cfad8ff7b48e7eb892c275a5cf0a3e82" secp256k1
+sui keytool import "0xd463e11c7915945e86ac2b72d88b8190cfad8ff7b48e7eb892c275a5cf0a3e82" secp256r1
+
+# import the mneomonics with derivation path to keystore. 
+sui keytool import "$MNEMONICS" ed25519
+sui keytool import "$MNEMONICS" secp256k1
+sui keytool import "$MNEMONICS" secp256r1
+```
+
+Create a transfer transaction in CLI. Set the $SUI_ADDRESS to the one corresponding to the key pair used to sign. Then sign with the private key corresponding to the sender address. 
+
+```shell
+sui client gas
+sui client transfer-sui --to $SUI_ADDRESS --sui-coin-object-id $GAS_COIN_ID --gas-budget $GAS_BUDGET --serialize-unsigned-transaction
+sui keytool sign --address $SUI_ADDRESS --data $TX_BYTES
+sui client execute-signed-tx --tx-bytes $TX_BYTES --signatures $SERIALIZED_SIGNATURE
+```
+
+</TabItem>
+</Tabs>
+
+### Notes
+1. This guide demonstrates how to sign with a single private key. Refer to [Multisig](/concepts/cryptography/transaction-auth/multisig.mdx) when it is preferred to set up more complex signing policies.
+2. Similarly, native zkLogin does not follow the above steps, see [zkLogin](/concepts/cryptography/zklogin.mdx) to understand how to derive a zkLogin address, produce a zkLogin signature with an ephemeral key pair. 
+3. If you decide to implement your own signing mechanisms instead of using the previous tools, see [Sui Signatures](/concepts/cryptography/transaction-auth/signatures.mdx) on the accepted signature specifications for each scheme. 
```

Comment:
```suggestion
3. If you decide to implement your own signing mechanisms instead of using the previous tools, see the [Signatures](/concepts/cryptography/transaction-auth/signatures.mdx) doc on the accepted signature specifications for each scheme. 
```